packages/patternfly-4/react-core/src/components/Breadcrumb/BreadcrumbItem.d.ts

```diff
@@ -0,0 +1,14 @@
+import { SFC, HTMLProps, ReactNode, ReactType } from 'react';
+
+export interface BreadcrumbItemProps extends HTMLProps<HTMLDivElement> {
+    children?: ReactNode;
+    className?: string;
+    to?: string;
+    isActive?: boolean;
```

Comment:
isActive is used to indicate the current page that the user is on within the breadcrumb.  How does this work with the BreadcrumbHeading.  The expectation is that the BreadcrumbHeading is the same as the current active BreadcrumbItem.

Reply:
Here's an example from PF3

http://patternfly-react.surge.sh/patternfly-3/index.html?selectedKind=patternfly-react%2FNavigation%2FBreadcrumb&selectedStory=Breadcrumb&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

Reply:
BreadcrumbHeading is always 'active'.

The main difference is that the title heading in PF3 is controlled by CSS on the breadcrumb list => you set title class on the breadcrumb list and the active item will inherit different attributes.

In PF-next you set the title heading class only on `<h>` element in the last child. (And the structure of the heading and a regular item is also different.)

If we want to set the title prop on the list, we will have to add some `breadcrumbContext` to share props and change rendering logic in the breadcrumbItem to something more difficult to render and read. I think it would be too unnecessarily complicated for a list. Better solution would be make changes in CSS to be the same as in PF3.

Reply:
Can we remove isActive then since it does the same thing as the title.

Reply:
I would remove this then since it's the same the Breadcrumb title.